### PR TITLE
Add Multi-Round Dialogue Function

### PR DESCRIPTION
**Pull request by cd9807**
In order to build a chat method that is more in line with public habits, I modified the current single round conversation to multiple rounds of conversation, and deleted Input, leaving only Instruction. I look forward to your adoption~

### CODE REVIEW

**Comment by tloen** [comment]
Agree with @AngainorDev. I do think an extra tab running something like the [Alpaca-LoRA-Serve](https://github.com/deep-diver/Alpaca-LoRA-Serve) could be an interesting addition. Generally speaking, though, inference isn't really a first-class citizen in this repo, given that a variety of inference frontends have already been built elsewhere. My main goal is to make finetuning as quick and easy as possible, and to provide the tools to export the finetuned weights to other services (alpaca-lora-serve, llama.cpp, etc.)

**Comment by cd9807** [comment]
Hi, thank you very much for your comments. I also agree with you two, but this time I want to exchange another question. When I tested the fine-tuned model, I found that duplicate characters were very serious. I wanted to consider the solution from both decoding and loss function perspectives. At that time, I found that there was no appropriate way to formulate the loss function, so I added the CustomTrainer class. Looking forward to your reply again~
